Jenkinsfile

```diff
@@ -2,12 +2,24 @@
 node {
   stage('checkout sources') {
         // You should change this to be the appropriate thing
-        git url: 'https://github.com/jschmersal-cscc/special-topics-labs-quality'
+        git url: 'https://github.com/Felixda1/special-topics-labs-ci.git'
   }
 
   stage('Build') {
-    // you should build this repo with a maven build step here
-    echo "hello"
+    withMaven (maven: 'maven3') {
+              sh "mvn package"
+            }
+  }
+  stage('Test') {
+              steps {
```

Comment:
I don't think you need this step (and I think it will cause problems actually).  On line 10 you do a `mvn package` which already runs the test.  Having your junit stanza should be good enough to create a report.